templates/ui_patterns_library/patterns-meta-information.html.twig

```diff
@@ -12,7 +12,7 @@
   {% set _rows = [] %}
   {% for field in pattern.fields %}
     {% set _rows = _rows|merge([
-      { cells: [
+      { data: [
```

Comment:
We need to do this change in `patterns-variant-meta-information.html.twig` as well.